agent/util-scripts/pbench-tool-meister-start

```diff
@@ -470,31 +1017,34 @@ def main(_prog, cli_params):
     # listen for responses from the Tool Data Sink.
     try:
         to_client_channel = f"{cli_tm_channel_prefix}-{tm_channel_suffix_to_client}"
-        redis_server = redis.Redis(host=redis_host, port=redis_port, db=0)
-        to_client_chan = RedisChannelSubscriber(redis_server, to_client_channel)
+        redis_client = redis.Redis(host=redis_server.host, port=redis_server.port, db=0)
+        to_client_chan = RedisChannelSubscriber(redis_client, to_client_channel)
```

Comment:
Having a variable named `to_client_chan` is too similar to the previously named `to_client_channel`.  I recommend renaming `to_client_channel` to `to_client_channel_name` (or, perhaps, just `channel_name`, or even `name`, since it's in a highly-restricted scope).